client/package.json

```diff
@@ -12,9 +12,11 @@
     "@fec/remark-a11y-emoji": "^3.1.0",
     "@geist-ui/core": "^2.3.5",
     "@geist-ui/icons": "^1.0.1",
+    "@types/cookie": "^0.4.1",
     "@types/js-cookie": "^3.0.1",
     "client-zip": "^2.0.0",
     "comlink": "^4.3.1",
+    "cookie": "^0.4.2",
```

Comment:
is it possible to use `js-cookie` to avoid adding another dependency? (or replacing js-cookie with cookie).

Reply:
hm, doesn't seem like it.